Casks/musescore.rb

```diff
@@ -46,7 +46,7 @@
     "~/Library/Caches/org.musescore.MuseScore",
     "~/Library/Preferences/org.musescore.MuseScore.plist",
     "~/Library/Preferences/org.musescore.MuseScore3.plist",
-    "~/Library/Preferences/org.musescore.MuseScore4.plist",
+    "~/Library/Preferences/org.musescore.MuseScore#{version.major}.plist",
```

Comment:
Even better might be to combine both with a wildcard

```suggestion
    "~/Library/Preferences/org.musescore.MuseScore*.plist",
```

Reply:
Seems quite fair.